recipe/patches/0001-Fix-relative-import.patch

```diff
@@ -0,0 +1,27 @@
+From 909d99c3d9a72571c0d6b7aee89fdcbc7ca8503b Mon Sep 17 00:00:00 2001
+From: Eric Lundby <Eric.Lundby@gmail.com>
+Date: Wed, 2 Apr 2025 14:04:43 -0600
+Subject: [PATCH 1/1] Fix-relative-import
+
+float8.h is next to mxfloat.h, just include it. 
+
+---
+ ml_dtypes/include/mxfloat.h | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/ml_dtypes/include/mxfloat.h b/ml_dtypes/include/mxfloat.h
+index 252e835..50fb8f0 100644
+--- a/ml_dtypes/include/mxfloat.h
++++ b/ml_dtypes/include/mxfloat.h
+@@ -25,7 +25,7 @@ limitations under the License.
+ #include <cstdint>
+ #include <limits>
+ 
+-#include "include/float8.h"
```

Comment:
I wonder if this is papering over an issue about setting include paths correctly?